docs/develprocess.md

```diff
@@ -221,6 +221,72 @@ We chose to base our `qemu-dp` repository on a fork of the upstream `qemu` [repo
   * If we really need to diverge a little from upstream, on a temporary dev branch `dev-NEXTXCPNGVERSION` (e.g. `dev-8.3`), based either on the current maintenance branch, or on a branch that is likely to be the one used in the next release (we don't always know!)
   * Next maintenance branch directly (e.g. `2.12.0-8.3`) once the upstream SRPMs have been released
 
+#### Special case: `host-installer`
+
+`host-installer` has an upstream repo, and we have a large number of
+modifications that have not yet made it upstream.  Changes are
+organized for best upstreaming them, which means using per-topic
+branches based on an upstream revision: those can then be used for an
+upstream PR, and for merging into an XCP-ng release.  Constructing an
+XCP-ng `host-installer` release then is done by starting from the
+upstream tag used by the matching XenServer release, and merging those
+topic branches.
+
+A `small-patches` branch is used to hold small non-upstreamable
+patches until we have decided what should be proposed upstream to
+replace them.  This one is merged first (as essentially made of older
```

Comment:
We don't use double space.
```suggestion
patches until we have decided what should be proposed upstream to
replace them. This one is merged first (as essentially made of older
```

Reply:
Damned we need a checker for this, I've been hardwired decades ago to use doublespaces...

Reply:
IIRC we started to work on something like that but it still is a WIP :thinking: 
But yeah a little checker wouldn't hurt!